@navikt/core/react/src/util/i18n/locales/en.ts

```diff
@@ -70,4 +72,32 @@ export default {
     charsTooMany: "{chars} characters too many",
     charsLeft: "{chars} characters left",
   },
+  Timeline: {
+    dayFormat: "d MMM",
+    monthFormat: "yyyy-MM",
```

Comment:
Wasn't sure how to format day+month and month+year to avoid ambiguity. This was the best I could come up with. Do you have any better ideas? Maybe I'm overthinking it and the context is enough to avoid ambiguity?